functests/10-test-conditions-templates.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+SCRIPTPATH=$( dirname $(readlink -f $0) )
+source ${SCRIPTPATH}/testlib.sh
+
+[ -z ${KV_NAMESPACE} ] && exit 127
+
+RET=1
+TEST_NS="${KV_NAMESPACE}"
+
+oc create -n ${KV_NAMESPACE} -f "${SCRIPTPATH}/common-templates-versioned-cr.yaml" || exit 2
+
+wait_for_deployment_ready ${TEST_NS} 5 50 "KubevirtCommonTemplatesBundle" "Running" "Successful"
+
+#wait for ssp operator to set proper conditions
+wait_for_condition ${TEST_NS} 5 20 "KubevirtCommonTemplatesBundle" "Available" "True"
```

Comment:
do we want to add a test that *some* templates are actually there? Something like template count > 1. That should cover all corners.

Reply:
We could do that too, but we need to use the right label (template.kubevirt.io: base iirc) to avoid seeing all the openshift ones.